package.json

```diff
@@ -1,50 +1,68 @@
 {
   "name": "oberon-redux",
   "version": "0.2.1",
-  "description": "Update your redux state directly from your components. No reducers, no actions. ",
+  "description": "Update your redux state directly from your components. No reducers, no actions.",
   "main": "lib/index.js",
+  "types": "lib/index.d.ts",
   "scripts": {
-    "test": "npm run build && jest",
-    "build": "npm run build:clean && npm run build:lib && npm run build:flow",
+    "test": "jest",
+    "build": "npm run build:clean && npm run build:lib",
     "build:clean": "rimraf lib/",
-    "build:lib": "babel src --out-dir lib",
-    "build:flow": "flow-copy-source -v src lib",
-    "build:docs": "documentation build src/index.js -f md > api.md"
+    "build:lib": "tsc",
+    "build:docs": "documentation build lib/index.js -f md > api.md",
+    "precommit": "tslint --project ."
   },
   "author": "Richard van Willegen",
   "license": "MIT",
   "repository": {
-    "type" : "git",
-    "url" : "https://github.com/oberonamsterdam/oberon-redux"
+    "type": "git",
+    "url": "https://github.com/oberonamsterdam/oberon-redux"
   },
   "devDependencies": {
+    "@types/dot-prop": "^4.2.0",
+    "@types/jest": "^22.2.3",
+    "@types/node": "^10.1.3",
     "babel-cli": "^6.26.0",
```

Comment:
alle babel deps kunnen weg

Reply:
Als ik dat doe, gaan er dingen mis in de travis CI:

"ReferenceError: Unknown plugin "transform-es2015-modules-commonjs" specified in "base" at 0, attempted to resolve relative to "/home/travis/build/oberonamsterdam/oberon-redux/src/updaters""

Reply:
als je `npm test` handmatig draait op je machine, falen de tests dan ook?

Reply:
Nee. Dat gaat goed.